config/vufind/searches.ini

```diff
@@ -366,7 +366,7 @@ pattern[sort_first_indexed_desc] = "^first_indexed desc$"
 ;       Display full-text, public scans from the Open Library (OL) Subjects API.
 ;       [GET parameter] (default = "lookfor"), [limit] (default = 5),
 ;       [date filter] The name of a date filter (from facet settings) to apply to
-;           searches.  Defaults to "publishDate" if blank; set to "false" to disable.
+;           searches.  Defaults to "publishDateRange" if blank; set to "false" to disable.
```

Comment:
I don't see a change to the actual default in the OpenLibrarySubjects recommendation module (the purpose of which I'm admittedly a bit fuzzy on). Should we revert this change?

Reply:
Reverted.